xml/vt_guestfs.xml

```diff
@@ -429,6 +429,38 @@ virt-ls -a sles.qcow2 /tmp/
 .X11-unix
 data.tar</screen>
    </sect3>
+   <sect3 xml:id="sec.guestfs.tools.others.log">
+    <title><command>virt-log</command></title>
+    <para>
+     <command>virt-log</command> shows the log files of the virtual machine
+     disk image or named libvirt domain. If the package
+     <systemitem class="resource">guestfs-winsupport</systemitem> is installed
```

Comment:
`<systemitem class=resource>` is outdated, we use `<package>` these days. (Yes, still lots of old systemitems around).